open_academy/wizards/open_academy_wizard.py

```diff
@@ -0,0 +1,18 @@
+from odoo import models, fields
+
+
+class OpenAcademyWizard(models.TransientModel):
+    _name = "open.academy.wizard"
+    _description = "Quick registration of attendees to sessions"
+
+    session_ids = fields.Many2many('open.academy.session',
+                                   default=lambda self: self._default_sessions())
```

Comment:
See this https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#33idioms

```suggestion
    session_ids = fields.Many2many(
                                   'open.academy.session',
                                   default=lambda self: self._default_sessions())
```